### ISSUE TITLE
autocomplete inflected forms in the search box

### ISSUE THREAD

**Issue by dwhieb** [UAlbertaALTLab/cree-intelligent-dictionary#711]
While the user is searching, show autocompletions of possible inflected forms.

From our discussions, it sounds like this could be difficult / undesirable to implement. Opening this issue as documentation that this was considered, and so that we can record our eventual decision about it.

**Comment by aarppe** [comment]
Yes, we can consider what this would require, and why that wouldn't potentially work (well) now.
